aea_projection.py

```diff
@@ -545,20 +529,20 @@ def __init__(self, ra_0=0, dec_0=0, dec_1=-30, dec_2=30, **kwargs):
         def __rho__(self, dec):
             return np.sqrt(self.C - 2 * self.n * np.sin(dec * self.deg2rad)) / self.n
 
-
         def transform_non_affine(self, xy):
             x = xy[:,0]
             y = xy[:,1]
 
             rho = np.sqrt(x**2 + (self.rho_0 - y)**2)
+
+            # FIXME: use arctan2 instead?
```

Comment:
Yes! Citing from the USGS manual by Snyder: "use the ATAN2 Fortran function, if n is negative, the signs of x, y, and rho_0 ... must be reversed before inserting them into [the theta equation]".

I'm pretty sure you just copied that from me, so I need to fix that, too.